job_definitions/notify_suppliers_of_framework_clarification_questions_answers.yml

```diff
@@ -36,12 +36,22 @@
             condition: UNSTABLE_OR_WORSE
             predefined-parameters: |
               USERNAME=framework-question-and-answers
-              JOB=Notify suppliers of new ${FRAMEWORK_SLUG} clarification questions and answers {{ environment }}
+              JOB=Notify suppliers of new ${FRAMEWORK_SLUG} clarification questions and answers - {{ environment }}
               ICON=:frame_with_picture:
               STAGE={{ environment }}
               STATUS=FAILED
               URL=<${BUILD_URL}consoleFull|#${BUILD_NUMBER}>
               CHANNEL=#dm-2ndline
+          - project: notify-slack
+            condition: UNSTABLE_OR_WORSE
```

Comment:
Don't we want `SUCCESS` instead of `UNSTABLE_OR_WORSE` here?

Reply:
Yep! 🤦‍♂